android/src/main/java/com/svtek/reactnative/RNAndroidSharerModule.java

```diff
@@ -187,7 +216,7 @@ public void shareViaInstagramStories(String filePath, String attributionUrl, fin
 
       Activity activity = this.reactContext.getCurrentActivity();
       if (activity.getPackageManager().resolveActivity(intent, 0) != null) {
-        activity.startActivityForResult(intent, 0);
+          activity.startActivityForResult(intent, 0);
```

Comment:
Please do not include changes that are not related to this commit's topic (`fix: fb share method`)

Reply:
Fixed some lines with the Android Studio Reformat Code.